images/rpmbuild-centos7/rpmlintrc.toml

```diff
@@ -0,0 +1,3 @@
+#suppress warnings for invalid-url Source0
+
+Filters = ['W: invalid-url Source*']
```

Comment:
we won't be keeping centos7 images now and this should have been removed already

Reply:
Thanks @Mossman1215 just removed centos7 image